### PR TITLE
Additional case for 1921 issue

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check #[FORDTCN-11966](https://luxproject.luxoft.com/jira/browse/FORDTCN-11966)

This PR is **ready for internal review**

### Summary
Added case with empty array after unknown parameter

### ATF version
develop

### Changelog
- updated script for 1921 issue 

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua [hunk 1]
```diff
@@ -137,6 +137,25 @@ local function ptuUpdateFuncNotValid(tbl)
...
+--[[ @ptuFuncEmptyArray: update table for PTU with empty array after removing unknown parameter
+--! @parameters:
+--! tbl - table for update
+--! @return: none
+--]]
+local function ptuFuncEmptyArray(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
```
Comment (VjKlepikov) [inline test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua]:
```suggestion
        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" , "NONE"},
```

File: test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua [hunk 2]
```diff
@@ -137,6 +137,25 @@ local function ptuUpdateFuncNotValid(tbl)
...
+        parameters = { unknownParameter }
+      }
```
Comment (VjKlepikov) [inline test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua:155]: please add    
  SubscribeVehicleData = {
        hmi_levels = { "BACKGROUND", "FULL", "LIMITED", "NONE" },
        parameters = { unknownParameter }
      }

Comment (VjKlepikov) [reply test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua:155]: Seems HMILevel for application is "NONE"

**Review by VjKlepikov — CHANGES_REQUESTED** [review]

**Comment by GetmanetsIrina** [comment]
@VjKlepikov, please review

**Review by VjKlepikov — APPROVED** [review]

**Comment by GetmanetsIrina** [comment]
moved to open

**Pull request closed without merging**